src/seals/util.py

```diff
@@ -51,17 +52,21 @@ def __init__(
 
         self.mask = np.ones(env.observation_space.shape, dtype=bool)
         for r in score_regions:
-            self.mask[r["x0"] : r["x1"], r["y0"] : r["y1"]] = 0
+            assert r["x"][0] < r["x"][1] and r["y"][0] < r["y"][1]
```

Comment:
Since this is a public method (users could create their own wrapper beyond our internal usage for seals-defined environments) you should raise a ValueError instead (and add a corresponding test). Passing in the wrong values is very much a possibility. 

`assert` is to be used when something should always behave in a certain way by virtue of the purported logic of the program. This allows catching logical bugs and reassuring code checkers. This would be fine if only our own internally defined masks could ever be used. However, when something is part of the public API and therefore contingent on user input, we cannot really _assert_ that a user won't pass the wrong value. See https://stackoverflow.com/questions/17530627/python-assertion-style#:~:text=The%20assert%20statement%20should%20only,user%20input%20or%20the%20environment. and https://wiki.python.org/moin/UsingAssertionsEffectively

What you _can_ do, however, is have tests that assert that our internally defined masks verify this. That, plus a `with raises` test on the MaskScoreWrapper API should be enough to thoroughly test this.

Reply:
Thanks for the explanation!